ui/pages/confirm-add-suggested-nft/confirm-add-suggested-nft.js

```diff
@@ -240,6 +240,7 @@ const ConfirmAddSuggestedNFT = () => {
                   return (
                     <Box
                       className="confirm-add-suggested-nft__nft-single"
+                      key={`confirm-add-suggested-nft__nft-single-${id}`}
```

Comment:
<img width="1091" alt="Screenshot 2023-10-12 at 11 18 22 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/c298c927-efdb-4e28-8524-bcce82ec7f6c">

Fixed this during the quest to fix CI/CD tests.